src/zenml/zen_stores/sql_zen_store.py

```diff
@@ -850,6 +852,19 @@ def migrate_database(self) -> None:
                 self.alembic.stamp(ZENML_ALEMBIC_START_REVISION)
                 self.alembic.upgrade()
 
+        # If an alembic migration took place, all non-custom flavors are purged
+        #  and the FlavorRegistry recreates all in-built and integration
+        #  flavors in the db.
+        revisions_afterwards = self.alembic.current_revisions()
```

Comment:
I'm not sure how complicated it is to test this, but I think it would be quite critical to make sure this works as intended?

Reply:
My worry as well, this also relies on the fact, that with every release we create an alembic revision

Reply:
I am not sure how to write these tests here, I think in general we need to write some testts, that our alembic migrations work as intended. It would then slot in well there